src/helm/benchmark/static/json-urls.js

```diff
@@ -1,24 +1,28 @@
 ////////////////////////////////////////////////////////////
 // Helper functions for getting URLs of JSON files
 
-function summaryJsonUrl(suite) {
-  return `${BENCHMARK_OUTPUT_BASE_URL}/runs/${suite}/summary.json`;
+function runManifestJsonUrl(release) {
```

Comment:
Likewise, I think we want these to be reverse compatible i.e. load from `/suite/` in reverse compatibility mode